tests/test_app.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+
+from starlette.testclient import TestClient
+
+from rapidhtml import RapidHTML
+from rapidhtml.tags import Html, H1, Div
+from rapidhtml.utils import get_default_favicon
+
+
+@pytest.fixture
+def app():
+    return RapidHTML()
+
+
+@pytest.fixture
+def client(app):
+    @app.route("/")
+    async def homepage(request):
+        return Html(
+            Div(
+                H1("foobar"),
+            )
+        )
+
+    return TestClient(app)
+
+
+def test_app(client):
+    response = client.get("/")
+    assert response.status_code == 200
+    assert "<title>RapidHTML</title>" in response.text
+    assert "<h1>foobar</h1>" in response.text
+    assert "htmx.org" in response.text
+
+
+def test_favicon(client):
+    response = client.get("/favicon.ico")
+    assert response.status_code == 200
+    assert response.headers["content-type"] == "image/svg+xml"
+    assert response.content == get_default_favicon()
```

Comment:
I am not sure myself, is this the best way to compare two large byte strings in python? :sweat_smile:

Reply:
Best way? Likely not... For the purposes of a unittest I think it works. Though I would like to know a better option for sure

I mostly just want to make sure the correct file is being passed. Could probably use a better test